src/silx/io/fioh5.py

```diff
@@ -166,6 +166,20 @@
     "BOOLEAN": "?",
 }
 
+def _bytestobool (val):
+    """Convert a byte of truth to true (1) or false (0).
+    True values are 'y', 'yes', 't', 'true', 'on', and '1'; false values
+    are 'n', 'no', 'f', 'false', 'off', and '0'.  Raises ValueError if
+    'val' is anything else.
+    """
```

Comment:
No need for a complete description in the docstring IMO, the code is self-explanatory


```suggestion
    """Convert bytes of a truth value to bool.

    Raises ValueError if 'val' is not supported.
    """
```